specs/language/conversions.tex

```diff
@@ -130,6 +155,37 @@
 prvalue of type \texttt{matrix<T,x,y>}. The destination value is the source
 value replicated into each element of the destination.
 
+\p A prvalue of type \texttt{vector<T,1>} can be converted
+to a prvalue of type \texttt{vector<T,x>}. The destination value is the value in
+the source vector replicated into each element of the destination.
+
+\Sec{Array and Class splat conversion}{Conv.asplat}
+
+\p A prvalue of type \texttt{T} can be converted to a
+prvalue of type \texttt{struct S}, if there are valid conversions from
+\texttt{T} to each \texttt{U} in flattened \texttt{S}, see section
+\ref{Decl.init.Agg} for description of a flattened ordering. The destination
```

Comment:
`\ref{Decl.init.Agg}` is resolving to `???` in the rendered document

Reply:
Could be that section isn't defined or their is a problem in the build process; You really have to run pdflatex like 2+ times. I'll check which.

Reply:
I confirmed it is a problem with the build process not the .tex file itself.